lib/rspec/core/output_wrapper.rb

```diff
@@ -3,13 +3,17 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        @output.respond_to?(name, priv)
+      end
```

Comment:
Why is this needed?  We're not using `method_missing` here. All the `output` methods that we proxy are defined at load time below via `define_method`.  So actually, I think this creates a bug: if `@output` is a kind of IO object that has extra methods not defined as an `IO` instance method (such as if `@output` is a `StringIO`--in that case it has an extra `string` method), then this will make it claim it responds to the extra methods, but will not proxy them because we haven't used `define_method` for them.